frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/helper-text-strings.ts

```diff
@@ -823,36 +890,29 @@ const helperTextStrings: HelpTextConfig[] = [
     name: "now",
     structure: "now",
     description: getDescriptionForNow,
-    example: "now",
     args: [],
   },
   {
     name: "convert-timezone",
-    structure:
-      "convertTimezone(" +
-      t`column` +
-      ", " +
-      t`target` +
-      ", [" +
-      t`source` +
-      "])",
+    structure: `convertTimezone(${argsPlaceholder})`,
```

Comment:
Would it be possible to remove `argsPlaceholder` at all? We could make `structure` contain only the user visible name of the function (`convertTimezone` in this case)